### ISSUE TITLE
[Packaging] Missing Protobuf dependency for Gramine-SGX packages

### ISSUE THREAD

**Issue by dimakuv** [gramineproject/gramine#269]
## Description of the problem

Following https://gramine.readthedocs.io/en/latest/quickstart.html to install the Gramine package.

When running the HelloWorld SGX example, we encounter the following error:
```
~/gramine/CI-Examples/helloworld$ gramine-sgx helloworld
/usr/lib/x86_64-linux-gnu/gramine/sgx/loader: error while loading shared libraries: \
        libprotobuf-c.so.1: cannot open shared object file: No such file or directory
```

## Actual results

It looks like our Gramine package dependencies miss the Google Protobuf libraries.
For now, one can circumvent this issue as follows: install the missing dependencies by hand, following https://gramine.readthedocs.io/en/latest/devel/building.html#required-packages. You can notice a couple packages that are related to `libprotobuf-c.so.1`:
- `libprotobuf-c-dev` 
- `protobuf-c-compiler` 
- `python3-protobuf`
 
Basically, the system needs these three packages to be installed, and then Gramine-SGX works correctly.
